app/presenters/link_check_csv_presenter.rb

```diff
@@ -0,0 +1,21 @@
+require 'csv'
+
+class LinkCheckCSVPresenter
+  def self.homepage_links_status_csv
+    CSV.generate do |csv|
+      csv << %w(status count)
+      LocalAuthority.group(:status).count.each do |key, value|
+        csv << [key || "nil", value]
+      end
+    end
+  end
+
+  def self.links_status_csv
+    CSV.generate do |csv|
+      csv << %w(status count)
+      Link.group(:status).count.each do |key, value|
```

Comment:
Could we scope this to only include links for services which are enabled? All other links aren't checked at all, so it's harder to work out how many links should be checked but haven't been yet for some reason (and so still have `nil` as their status) if we include all the links we'll never check.